### PR TITLE
Consolidates project

### PR DESCRIPTION
**Pull request by adriancole**
This is the real work after #14, which makes the ingester a single project under a single group ID. This does not result in different docker images.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
+  pluginManager.withPlugin("org.hypertrace.publish-plugin") {
+    configure<org.hypertrace.gradle.publishing.HypertracePublishExtension> {
+      license.set(org.hypertrace.gradle.publishing.License.TRACEABLE_COMMUNITY)
+    }
+  }
+
+
+//  pluginManager.withPlugin("org.hypertrace.publish-plugin") {
+//    configure<HypertracePublishExtension> {
+//      license.set(License.APACHE_2_0)
```
Comment (adriancole) [inline build.gradle.kts:23]: we had one project here which was formerly ASL. @aaron-steinfeld do you happen to know the syntax to make an exception per-project? If so, feel free to push a commit to do that!

File: raw-spans-grouper/build.gradle.kts [hunk 2]
```diff
@@ -1,20 +1,55 @@
 plugins {
```
Comment (adriancole) [inline raw-spans-grouper/build.gradle.kts:1]: raw-spans-grouper was a repo with only one source base

File: build.gradle.kts (hunk 1 above)
Comment (adriancole) [reply build.gradle.kts:23]: something subtle about the following isn't working.. would be easier if this were property driven

```kotlin
subprojects.forEach {
  group = "org.hypertrace.ingester"

  pluginManager.withPlugin("org.hypertrace.publish-plugin") {
    // span-normalizer historically had a different license
    val licenseVal = if (it.projectDir.getName() == "span-normalizer")
      License.APACHE_2_0 else License.TRACEABLE_COMMUNITY

    configure<HypertracePublishExtension> {
      license.set(licenseVal)
    }
  }
```

Comment (adriancole) [reply build.gradle.kts]: nm it was really only one module

**Comment by adriancole** [comment]
@kotharironak if you can test this branch we can tell if you feel it is safe to merge. After that, we would merge commits in #14 and squash commits here

**Comment by kotharironak** [comment]
@adriancole I will try this out for #14

**Comment by adriancole** [comment]
cleared my docker system and rebuild images with gradle.. looks like it builds them all:

```
REPOSITORY                             TAG                 IMAGE ID            CREATED             SIZE
hypertrace/span-normalizer             test                ad8de3af5cca        27 seconds ago      151MB
hypertrace/raw-spans-grouper           test                ec8e25736b45        27 seconds ago      150MB
hypertrace/hypertrace-ingester         test                887351e908a0        27 seconds ago      205MB
hypertrace/hypertrace-view-generator   test                a59ff15ba0c2        27 seconds ago      192MB
hypertrace/hypertrace-trace-enricher   test                559592984fc1        27 seconds ago      201MB
hypertrace/hypertrace-view-creator     test                ebe84b7e066e        27 seconds ago      238MB
hypertrace/java                        11                  82d8db2682c9        3 weeks ago         102MB
```

What I've not done is change the circleci config to accomodate multiple helm charts. the existing yaml seems to expect a single variable and I'm not sure the way circle does matrixes. OTOH, I think anyone familiar with circle can do a matrix for the existing helm charts.. or consolidate them in a different commit.

Will wait for another to finish this iotw.

**Comment by adriancole** [comment]
snyk is red on over quota. I presume people know about this.

**Comment by adriancole** [comment]
re-pushed

File: .circleci/helm.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+cd $ROOT_PROJECT_DIR
+SUB_PROJECTS_DIRS=$(find . -iname "helm" | sed 's/\(.*\)\/.*/\1/')
+
+script=$0
+subcommand=$1; shift
+
+case "$subcommand" in
+  validate)
+    for SUB_PROJ_DIR in $SUB_PROJECTS_DIRS; do
+      cd $SUB_PROJ_DIR
```
Comment (jcchavezs) [inline .circleci/helm.sh:15]: Do we need to cd the `$SUB_PROJ_DIR`, can't we just run

```bash
helm dependency update $SUB_PROJ_DIR/helm/
helm lint --strict $SUB_PROJ_DIR/helm/
helm template $SUB_PROJ_DIR/helm/ --output-dir $SUB_PROJ_DIR/helm/
```

**Comment by kotharironak** [comment]
validate-charts is failing as it seems `helm-gcs-packager` is not supporting bash script

**Comment by kotharironak** [comment]
@adriancole Works fine locally if I run `sh .circleci/helm.sh`, seems like missing something.

**Comment by kotharironak** [comment]
@adriancole @JBAhire @jcchavezs This PR is mostly ready except for one more time validation. Pl. have a look. 

After validating this PR, the order in which PRs needs to merge in order:
the disable-publish workflow in the individual repo:
https://github.com/hypertrace/raw-spans-grouper/pull/36
https://github.com/hypertrace/raw-spans-grouper/pull/36
https://github.com/hypertrace/hypertrace-trace-enricher/pull/48
https://github.com/hypertrace/hypertrace-view-generator/pull/32

Once above merged:
https://github.com/hypertrace/hypertrace-ingester/pull/15

**Comment by adriancole** [comment]
It takes almost no time for the prepared merge to take place. Adding additional worksteps to keep the dead repos alive is only going to encourage divergence. I'd recommend not doing the bookkeeping on the dead repos and instead just archive them.

**Review by JBAhire — APPROVED** [review]

**Comment by jcchavezs** [comment]
<3

**Pull request merged**